app/views/shared/_new_room_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <div id="new-room-form">
+    <div class="input-field col s6">
+      <h5>Add a new topic room</h5>
+      <%= form_for :room, url: rooms_path do |f| %>
+        <%= f.text_field :name, required: true, placeholder: "Topic: ex. testing, quotations, workflow" %>
+        <%= f.text_field :choice, placeholder: "Choice: ex. minitest, rspec, mrspec" %>
+        <%= current_user.form_link(f) %>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
Yeah I think I said in slack that after playing with the app having an entire view for creating a room is unnecessary. I only wanted to keep this one in for the PR so I could show off how to use Draper to dynamically set the f.submit in a form_for.